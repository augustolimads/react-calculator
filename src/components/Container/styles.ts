import styled from 'styled-components'

export const Wrapper = styled.main`
  height: 100vh;
  width: 100%;
  background-image: url('img/beautiful-bg.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 0% 100%;
  display: flex;
  justify-content: center;
  align-items: center;
`
