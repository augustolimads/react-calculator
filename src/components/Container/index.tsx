import * as S from './styles'

const Container = () => (
  <S.Wrapper>
    <span>Container</span>
  </S.Wrapper>
)

export default Container
