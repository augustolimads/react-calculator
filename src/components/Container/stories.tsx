import { Story, Meta } from '@storybook/react/types-6-0'
import Container from '.'

export default {
  title: 'Container',
  component: Container
} as Meta

export const Default: Story = () => (
  <div style={{ maxWidth: '130rem', margin: '0 auto' }}>
    <Container />
  </div>
)
