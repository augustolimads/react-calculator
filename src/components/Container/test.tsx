import { render, screen } from '@testing-library/react'

import Container from '.'

const props = {
  children: 'testando container'
}

describe('<Container />', () => {
  it('should render the children content', () => {
    const { container } = render(<Container {...props} />)
    expect(screen.getByText('testando container')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
