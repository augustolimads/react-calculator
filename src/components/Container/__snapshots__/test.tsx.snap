// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Container
  </h1>
</main>
`;
